ESP8266/ESP8266.cpp

```diff
@@ -832,10 +840,29 @@ void ESP8266::_oob_watchdog_reset()
         _sock_i[i].open = false;
     }
 
-    _conn_status = NSAPI_STATUS_DISCONNECTED;
+    // Makes possible to reinitialize
+    _conn_status = NSAPI_STATUS_ERROR_UNSUPPORTED;
     _conn_stat_cb();
 }
 
+void ESP8266::_oob_busy()
+{
+    char status[5];
+    if (_parser.recv("%4[^\"]\n", status)) {
+        if (strcmp(status, " s...\n") == 0) {
+            ; //TODO maybe do something here, or not...
```

Comment:
What is the whole reason for this OOB handler if it does not do anything?

Reply:
Just making people aware there are this kind of events which might be raised. And especially to report an error if some new firmware introduces new messages on which we are to react.